cypress/e2e/charging_stations.cy.ts

```diff
@@ -0,0 +1,39 @@
+describe('Charging station actions', () => {
+  beforeEach(() => {
+    cy.clearAllCookies();
+    cy.clearAllLocalStorage();
+    cy.clearAllSessionStorage();
+
+    //TODO figure out why this is thrown!
+    const stub = cy.stub();
+    Cypress.on('uncaught:exception', (err, runnable) => {
+      if (err.message.includes('ResizeObserver')) {
+        stub();
+        return false;
+      }
+    });
+
+    cy.visit('http://localhost:5173/charging-stations');
+  });
+
+  it('Get Charging station actions', () => {
+    cy.getByData('citrine-os-icon').should(
+      'have.attr',
+      'src',
+      '/Citrine_OS_Logo.png',
+    );
+
+    cy.getByData('custom-action-dropdown-button').click();
+    cy.get('ul.ant-dropdown-menu[role="menu"] li[role="menuitem"]')
+      .eq(11) // 0-based index, so 11 corresponds to the 12th item
+      .click();
+    cy.getByData('field-type-input').click();
+    cy.get('.ant-select-dropdown')
+      .should('not.have.class', 'ant-select-dropdown-hidden')
+      .and('be.visible');
+    cy.getByData('field-type-input-option-Immediate')
+      .should('be.visible')
+      .click();
+    cy.getByData('ResetData2-generic-form-submit').click();
```

Comment:
you dont need to also select an evse before submitting?

Reply:
also, I think we should add something that check for a `notification` to pop up that says that the message went through successfully, do you think that would be feasible? 🙏  

Reference:
```
export const triggerMessageAndHandleResponse = async <T,>(
  url: string,
  responseClass: Constructable<T>,
  data: any,
  responseSuccessCheck: (response: T) => boolean,
) => {
  try {
    const client = new BaseRestClient();
    const response = await client.post(url, responseClass, {}, data);

    // todo reuse handle response!
    if (responseSuccessCheck(response)) {
      showSucces();
      ...
```

and

```
export const showSucces = () => {
  notification.success({
    message: 'Success',
    description: 'The request was successful.',
    placement: 'topRight',
  });
};
```

Reply:
EVSE isn't required since you can reset the entire chargebox.

yes, this was next on my list to dig deeper into.

Reply:
Dug a bit deeper. Added checking for the notification and added a `data-test` prop to the notification. 
To avoid having to actually hook up a charger for the test I intercept the http request.